ceph.yaml

```diff
@@ -0,0 +1,344 @@
+package:
+  name: ceph
+  version: 19.2.1
+  description: Distributed object, block, and file storage
+  copyright:
+    - license: LGPL-2.1
+  dependencies:
+    runtime:
+      - curl
+      - libaio
+      - libedit
+      - libgcc
+      - libstdc++
+      - libudev
+      - openssl
+      - py${{vars.py-version}}-bcrypt-3.2
+      - py${{vars.py-version}}-cherrypy
+      - py${{vars.py-version}}-cryptography
+      - py${{vars.py-version}}-jinja2
+      - py${{vars.py-version}}-jsonpatch
+      - py${{vars.py-version}}-packaging
+      - py${{vars.py-version}}-pecan
+      - py${{vars.py-version}}-prettytable
+      - py${{vars.py-version}}-pyopenssl
+      - py${{vars.py-version}}-python-dateutil
+      - py${{vars.py-version}}-pyyaml
+      - py${{vars.py-version}}-requests
+      - py${{vars.py-version}}-scipy
+      - py${{vars.py-version}}-urllib3
+      - py${{vars.py-version}}-werkzeug
+      - python-${{vars.py-version}}
+      - yaml
+      - zlib
+
+vars:
+  py-version: 3.11
+
+environment:
+  contents:
+    packages:
```

Comment:
Looking at the build logs, it seems we need to have: `bzip2`, `dot` (for doxygen component)

Reply:
Ignore me, doxygen should provide this at runtime level anyway.